docs/docs/APIdocs/layers/nn/Convolution Layers/SpatialConvolution.md

```diff
@@ -0,0 +1,157 @@
+## SpatialConvolution ##
+
+**Scala:**
+```scala
+val m = SpatialConvolution[T](nInputPlane,nOutputPlane,kernelW,kernelH,strideW,strideH,padW,padH,nGroup,propagateBack,wRegularizer,bRegularizer,initWeight, initBias, initGradWeight, initGradBias)
+```
+**Python:**
+```python
+m = SpatialConvolution(n_input_plane,n_output_plane,kernel_w,kernel_h,stride_w=1,stride_h=1,pad_w=0,pad_h=0,n_group=1,propagate_back=True,init_method="default")
+```
+
+SpatialConvolution is a module that applies a 2D convolution over an input image.
+
+The input tensor in `forward(input)` is expected to be
+either a 4D tensor (`batch x nInputPlane x height x width`) or a 3D tensor (`batch x height x width`). The convolution is performed on the last two dimensions.
```

Comment:
I think if input is 3D tenor, it should be nInputPlan(other than batch) x height x width